org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java

```diff
@@ -185,12 +186,29 @@ private void runAnalysisAndUpdateMarkers(@Nullable Server server, Map<ISonarLint
     Map<ISonarLintIssuable, List<Issue>> issuesPerResource = new LinkedHashMap<>();
     filesToAnalyze.keySet().forEach(slFile -> issuesPerResource.put(slFile, new ArrayList<>()));
 
+    long start = System.currentTimeMillis();
     AnalysisResults result = run(server, config, issuesPerResource, monitor);
     if (!monitor.isCanceled() && result != null) {
+      updateTelemetry(inputFiles, start);
```

Comment:
Is it intended to not count marker processing in the analysis time? IMO we should count everything

Reply:
Yes that's what I intended. I go with your opinion, I moved down the line.